builtin/providers/aws/resource_aws_dynamodb_table.go

```diff
@@ -158,6 +159,20 @@ func resourceAwsDynamoDbTable() *schema.Resource {
 					return hashcode.String(buf.String())
 				},
 			},
+			"stream_enabled": &schema.Schema{
+				Type: schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"stream_view_type": &schema.Schema{
+				Type: schema.TypeString,
+				Optional: true,
+				Computed: true,
```

Comment:
Should we add a ValidateFunc here so we can catch user typos at plan time? Or do you think the set of valid attributes will change frequently enough that we should avoid trying to duplicate the correct list of values here?